contracts/hardhat_overrides.ts

```diff
@@ -0,0 +1,162 @@
+export const overrides = {
```

Comment:
We should be able to extract different objects (L1 user-facing contract, L1 Verifier extends previous one with low compilation runs, L2 user-facing contracts) to reduce code duplication here